manifest.json

```diff
@@ -19,7 +26,7 @@
         "mysql"
     ],
     "requirements": {
-      "yunohost": ">= 4.1.0"
+      "yunohost": ">= 4.2.4"
```

Comment:
Why the required version bump?

Reply:
This doesn't require a version bump. But motivating users to upgrade to the last version of YunoHost should be encouraged as it corrects many bugs and security issues.

Reply:
Hmm. I understand how keeping Yunohost up-to-date is better for the user.

But on the other side, upgrading is not always a simple task. And there are legitimate reasons for wanting to upgrade a packaged app without upgrading the whole system. Also, using packaged apps as a way to force upgrades is a side-effect that doesn't make the intention clear (for instance it may instead cause app updates not to be visible in the web UI, if Yunohost is not upgraded enough). For me this is user-hostile, and goes against the liberty of the user to install and use the software however they want (provided there are no strong technical reasons to prevent the upgrade).

IMHO, if it is better for Yunohost to stay upgraded, then this should be handled or hinted by Yunohost itself (for instance by messages in the moulinette or in the web admin). But in this packaged app, I'd prefer to keep the required Yunohost version relatively low and permissible.

Reply:
```suggestion
      "yunohost": ">= 4.1.0"
```